theseus/embodied/measurements/tests/test_between.py

```diff
@@ -130,7 +130,7 @@ def test_error_between_se2():
 
 
 def test_jacobian_between_se3():
-    for batch_size in [1, 20, 100]:
+    for batch_size in [1, 10]:
```

Comment:
This [1, 10] actually shows up so often that I am wondering if we should add it to constants.py?

Reply:
Added a constant to `theseus.core.tests.common` and replaced other similar instances with this. Now all tests use `BATCH_SIZES_TO_TEST = (1, 10)`.